src/ontology/edits/oeo-shared.omn

```diff
@@ -1692,7 +1692,7 @@ https://github.com/OpenEnergyPlatform/ontology/pull/1360",
     SubClassOf: 
         OEO_00000200,
         <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000144,
-        <http://purl.obolibrary.org/obo/BFO_0000051> min 1 OEO_00000253
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000253
```

Comment:
Please document this change in the term tracker item of `electricity grid` by referencing the issue and the PR. In the term tracker item you might shortly describe the change (i.e. what you've currently put in CHANGELOG.md).